<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1055</width>
    <height>838</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Label Editor</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QGroupBox" name="video_groupBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>1677215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="title">
       <string/>
      </property>
      <layout class="QGridLayout" name="gridLayout_5"/>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QGroupBox" name="control_groupBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string/>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="3" column="12" colspan="2">
        <widget class="QPushButton" name="close_video_pushButton">
         <property name="text">
          <string>Close</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="label_8">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Bad Frame</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QDoubleSpinBox" name="current_bad_frame_doubleSpinBox"/>
       </item>
       <item row="1" column="3">
        <widget class="QLabel" name="label_3">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>/</string>
         </property>
        </widget>
       </item>
       <item row="1" column="4">
        <widget class="QDoubleSpinBox" name="total_bad_frame_doubleSpinBox">
         <property name="frame">
          <bool>true</bool>
         </property>
         <property name="readOnly">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="1" colspan="13">
        <widget class="QSlider" name="current_frame_horizontalSlider">
         <property name="maximum">
          <number>99</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="3" column="8">
        <widget class="QPushButton" name="set_dir_pushButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>20</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Video File Path</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2" colspan="6">
        <widget class="QLineEdit" name="video_file_path_lineEdit"/>
       </item>
       <item row="4" column="1">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Position Suffix</string>
         </property>
        </widget>
       </item>
       <item row="4" column="2" colspan="9">
        <widget class="QPlainTextEdit" name="position_suffix_plainTextEdit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>27</height>
          </size>
         </property>
         <property name="inputMethodHints">
          <set>Qt::ImhNone</set>
         </property>
        </widget>
       </item>
       <item row="3" column="9" colspan="3">
        <widget class="QPushButton" name="load_pushButton">
         <property name="text">
          <string>Load</string>
         </property>
         <property name="shortcut">
          <string>Ctrl+L</string>
         </property>
        </widget>
       </item>
       <item row="1" column="13">
        <widget class="QPushButton" name="next_frame_pushButton">
         <property name="text">
          <string>-&gt;</string>
         </property>
         <property name="shortcut">
          <string/>
         </property>
        </widget>
       </item>
       <item row="1" column="12">
        <widget class="QDoubleSpinBox" name="total_frame_doubleSpinBox">
         <property name="frame">
          <bool>true</bool>
         </property>
         <property name="readOnly">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="11">
        <widget class="QLabel" name="label">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>/</string>
         </property>
        </widget>
       </item>
       <item row="1" column="10">
        <widget class="QDoubleSpinBox" name="current_frame_doubleSpinBox"/>
       </item>
       <item row="1" column="9">
        <widget class="QLabel" name="label_7">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Frame</string>
         </property>
        </widget>
       </item>
       <item row="1" column="5">
        <widget class="QPushButton" name="next_bad_frame_pushButton">
         <property name="text">
          <string>-&gt;</string>
         </property>
         <property name="shortcut">
          <string>Alt+B</string>
         </property>
        </widget>
       </item>
       <item row="1" column="6">
        <widget class="QPushButton" name="extrapolate_pushButton">
         <property name="text">
          <string>Extrapolate</string>
         </property>
         <property name="shortcut">
          <string>Alt+B</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1" colspan="6">
        <widget class="QSlider" name="current_bad_frame_horizontalSlider">
         <property name="maximum">
          <number>99</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="7">
        <widget class="QPushButton" name="clear_pushButton">
         <property name="text">
          <string>clear</string>
         </property>
         <property name="shortcut">
          <string>Alt+C</string>
         </property>
        </widget>
       </item>
       <item row="4" column="12" colspan="2">
        <widget class="QProgressBar" name="progressBar">
         <property name="value">
          <number>0</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1055</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
